mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -157,6 +157,11 @@ def test_events():
     assert_raises(TypeError, read_raw_brainvision, vhdr_path, eog=eog,
                   preload=True, response_trig_shift=np.nan)
 
+    # Test that both response_trig_shit and event_id can be set
+    read_raw_brainvision(vhdr_path, eog=eog, preload=False,
+                         response_trig_shift=100,
+                         event_id={'Sync On': 5})
```

Comment:
To be clear, this is the one instance that should *not* raise any warnings, as it's neither conflating R and S items, nor dropping the Sync.